ethpm/deployments.py

```diff
@@ -1,5 +1,50 @@
-class Deployments:
+from ethpm.exceptions import ValidationError
+
+from ethpm.utils.contract import (
+    validate_contract_name,
+)
+
 
-    def __init__(self, deployment_data, w3):
+class Deployments:
+    """
+    Deployment object to access instances of
+    deployed contracts belonging to a package.
+    """
+    def __init__(self, deployment_data, contract_types, w3):
```

Comment:
Wasn't sure the best way to handle the contract_types, so for now just passing in the `"contract_types".keys()` from `package_data` for validation - if there's a better way to handle it, I can make the changes.